lib/improver/psychrometric_calculations/psychrometric_calculations.py

```diff
@@ -657,35 +662,78 @@ def find_falling_level(self, wb_int_data, orog_data, height_points):
         #  for falling_level_threshold so we take the 0 index
         snow_level_data = interpolate(np.array([self.falling_level_threshold]),
                                       wb_int_data, asl, axis=0)[0]
+
         return snow_level_data
 
-    @staticmethod
-    def fill_in_missing_data(snow_level_data):
+    def fill_in_missing_data(self, snow_level_data, orog_data,
+                             highest_wb_int_data, highest_height):
         """
-        Fill in missing data
+        Fill in missing data. Wet-bulb integral data
+        is only available above ground level and there may be sn insufficient
+        number of levels in the input data, in which case the falling_level
+        will have been set to np.nan. This function fills in the missing
+        data by firstly setting sea-level points.
```

Comment:
This feels incomplete. Please can you append snippets from the excellent in-line comments further down the code to this statement.